plugins/web-search/web-search.plugin.zsh

```diff
@@ -47,6 +48,7 @@ function web_search() {
 
 
 alias bing='web_search bing'
+alias br='web_search brave'
```

Comment:
Could you change that to `brave`? `br` might exist in people's path, could be a tipical alias for whatever!

Reply:
`brave` might be the command to start the browser. An alternative might be `brave_search`, which is not likely to override anything, but I'm not satisfied with it either.

Reply:
Made the changes, thank you for the feedback.